src/file-dropzone/index.tsx

```diff
@@ -0,0 +1,18 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+
+import useBaseComponent from '../internal/hooks/use-base-component';
+import { applyDisplayName } from '../internal/utils/apply-display-name';
+import { FileDropzoneProps } from './interfaces';
+import InternalFileDropzone from './internal';
+import { useFilesDragging } from './use-files-dragging';
```

Comment:
Can we add metrics instrumentation to useFilesDragging (as a follow-up, I'd say)?